src/Compilers/CSharp/Portable/Symbols/MemberSymbolExtensions.cs

```diff
@@ -192,6 +192,20 @@ internal static ImmutableArray<ParameterSymbol> GetParametersIncludingExtensionP
             return symbol.GetParameters();
         }
 
+        internal static int GetOrdinalIncludingExtensionParameter(this ParameterSymbol parameter)
+        {
+            int offset = 0;
+            Symbol containingSymbol = parameter.ContainingSymbol;
+            if (containingSymbol.GetIsNewExtensionMember()
+                && !containingSymbol.IsStatic
+                && parameter.ContainingType.ExtensionParameter is not null)
```

Comment:
>&& parameter.ContainingType.ExtensionParameter is not null [](http://example.com/codeflow?start=16&length=58)

Could this condition lead to an undesired behaviors in error scenarios. Like accessing wrong argument or parameter?

Reply:
Also, it looks like the helper is used only in one place and it is probably not worth having a dedicated method for it.